### DISCUSSION TITLE
A Layout Artist Wanted for The Bumpkin News!

### DISCUSSION THREAD

**Discussion by drumdrumdrum3337** [sunflower-land/sunflower-land#1487] [Art]
Do you believe there's just nothing better to go with your morning cup of Joe than a good old-fashioned newspaper? 
Do you know a thing or two about newspaper layouts? 
Does the idea of crafting a first in-game crypto newspaper in the world sounds fascinating to you?

If you have answered 'yes' to all 3, then read on!

The Bumpkin News team is looking for a layout artist for the newspaper ([initial proposal is here](https://github.com/sunflower-land/sunflower-land/discussions/1367))! We are still in the planning stages, which means your decisions will have a great impact on how the newspaper will look like once it is live!

Currently, the team includes Tiffany, who'll take care of the coding part (she's also part of the Project Dignity), and Jc as an artist.

If you'd like to join us on this journey through the uncharted territories of in-game media, please submit the following:

1. A bit about yourself: totally freestyle and include whatever you believe could be relevant. 
2. A mock-up of a page or two of the newspaper (a cover, or an inside page, or both). It should be:

- following the spirit of the game (as it will be read inside the game), but not necessarily sticking to the game's fonts and styles;
- should be able to accommodate large chunks of text that would be easily readable;
- should be able to accommodate art and pictures;
- should indeed remind a 'real-world' newspaper.

Pic just for illustration purposes, feel free to follow your own vision!
<img width="611" alt="Screen Shot 2022-10-05 at 8 31 54 am" src="https://user-images.githubusercontent.com/108594866/194281459-2334992a-4c8a-4399-a309-2e7d649b3b4e.png">

Important notes:
- the newspaper is planned to go out every two weeks, hence this might be a somewhat time consuming project;
- this project is not a volunteer-based one, it entails sharing of the profit generated from the sales of the newspaper between members of the team.
- feel free to post your submissions in this discussion, reach out on the discord, or just send us an email to the-bumpkin-news@googlegroups.com

Feel free to use (as well as not to) the following items
![the_bumpkin_news_official_logo_-big-](https://user-images.githubusercontent.com/108594866/194285334-54204e4f-b3c1-4d89-bae1-110f7fde76f1.png)
![bumpkin_newspaper_logo](https://user-images.githubusercontent.com/108594866/194285383-39e845f5-4e8b-4531-ab9b-f1006095fc35.png)

**Comment by Myth727** [comment]
Any updates on this? Will love to see a newspaper in game!

**Reply by drumdrumdrum3337** [reply]
Hey @Myth727 actually we were finally able to make some progress on the layout front! Stay tuned :)
